pallets/avn/src/lib.rs

```diff
@@ -206,6 +206,8 @@ impl<T: Config> Pallet<T> {
         let mut url = String::from("eth/sign/");
         url.push_str(data_to_sign);
 
+        info!(target: "avn-service", "avn-service sign request (ecdsa) for data {:?}", data_to_sign);
```

Comment:
Please, add the hex version of the encoded data if possible. That is the definitive information we need to have, not the raw human-readable data (that is useful for debugging, ofc, but it's the encoding that matters)

Reply:
it is the hex encoded data. I will add it to the log to make it more obvious.